mmpose/datasets/datasets/body3d/body3d_mview_direct_panoptic_dataset.py

```diff
@@ -211,7 +211,7 @@ def _get_db(self):
                             dtype=np.float32)
 
                         cnt = 0
-                        person_ids = -np.ones(self.max_persons, dtype=np.int)
```

Comment:
`np.int` was an alias to `int` and would be int64 in a 64-bit system. If so, should we keep this behavior?

Reply:
ok. let's use int instead.